base_layer/core/src/consensus/consensus_manager.rs

```diff
@@ -174,6 +190,17 @@ impl ConsensusManagerBuilder {
             network: self.network,
             emission,
             gen_block: self.gen_block,
+            chain_strength_comparer: self.chain_strength_comparer.unwrap_or(
+                strongest_chain()
+                    .by_accumulated_difficulty()
+                    .then()
+                    .by_monero_difficulty()
+                    .then()
+                    .by_blake_difficulty()
+                    .then()
+                    .by_height()
+                    .build(),
+            ),
```

Comment:
I have a suspicion this might increase the incentive for a selfish mine. Especially from the monero side as you know that if your difficulty is higher, it will be favoured more. 

I would put height as the fist tie breaker, then monero then blake. Height should break most ties and favour the chain that moves the block_chain to most forward in terms of mined.